charts/jaeger/README.md

```diff
@@ -262,6 +262,7 @@ The following table lists the configurable parameters of the Jaeger chart and th
 | `collector.service.annotations` | Annotations for Collector SVC | `nil` |
 | `collector.service.grpc.port` | Jaeger Agent port for model.proto | `14250` |
 | `collector.service.http.port` | Client port for HTTP thrift | `14268` |
+| `collector.service.loadBalancerIP` | IP Address to be assigned to the Load Balancer | `nil` |
```

Comment:
nit: "load balancer" does not require uppercase as they are common nouns in this case.

Maybe just reuse the description you put in the values file?

Reply:
Done :)